packages/@react-stately/color/src/useColorPickerState.ts

```diff
@@ -15,13 +15,13 @@ export interface ColorPickerState {
 
 export function useColorPickerState(props: ColorPickerProps): ColorPickerState {
   let value = useColor(props.value);
-  let defaultValue = useColor(props.defaultValue || '#0000')!;
+  let defaultValue = useColor(props.defaultValue || '#000000')!;
   let [color, setColor] = useControlledState(value || undefined, defaultValue, props.onChange);
   
   return {
     color,
     setColor(color) {
```

Comment:
- add if check for null to early return in here
- update useColorSwatch so that color is required
- update docs (RSP/RAC/hooks)